torch/csrc/distributed/rpc/request_callback_impl.cpp

```diff
@@ -151,14 +151,19 @@ std::unique_ptr<RpcCommandBase> RequestCallbackImpl::processRpc(
           autogradContainer.newAutogradMessageId());
 
       auto response = c10::guts::make_unique<RpcWithAutograd>(
+          rpc::RpcAgent::getDefaultRpcAgent()->getWorkerInfo().id_,
```

Comment:
this will throw unless we add a python `init_rpc_agent` that calls `setDefaultRpcAgent` in C++ as in https://github.com/pytorch/pytorch/pull/27022/

Reply:
We probably need to do this to get this PR landed correct?

Reply:
yup, this is added in this PR below.